packages/template-retail-react-app/jest.config.js

```diff
@@ -24,7 +24,8 @@ module.exports = {
         'scripts/generator/*.{js,jsx}',
         '!app/pages/test-container/**/*.{js,jsx}',
         '!app/utils/test-utils.js',
-        '!app/commerce-api/mocks/*js',
+        '!app/mocks/*.js',
+        '!app/commerce-api/**/*.js',
```

Comment:
We can remove this when we delete commerce-api folder. Excluding this increases overall coverage

Reply:
```suggestion
        // TODO: This is no longer necessary when the commerce-api folder is removed.
        '!app/commerce-api/**/*.js',
```
🤷